src/intimate-objects.md

```diff
@@ -0,0 +1,9 @@
+## Intimate Objects
+
+TABLEFLIP were comissioned by Intimate Objects to build a website and CMS allowing visitors to design, create and share intimate objects with each other.
+
+<div class="full">
```

Comment:
In general, I'd prefer to keep "content" as markdown where ever possible. In this case the benefit is less clear I realised I needed to add the `.full` div to keep the expando magic. My assumption at the time was we'd figure out another solution for the expando at some point, so the div could be removed from the markdown.